bendigobank.rb

```diff
@@ -0,0 +1,87 @@
+require 'watir'
+require 'json'
+require 'nokogiri'
+require 'date'
+require_relative 'account'
+require_relative 'transaction'
+
+class Bendigobank
+
+  def self.execute
+    connect
+    fetch_accounts(connect)
+    fetch_transactions(connect)
+  end
+
+  def self.connect
+    Watir.logger.ignore :deprecations
+    browser = Watir::Browser.new :firefox
```

Comment:
You can create variable 
`@browser = Watir::Browser.new :firefox`
with `@` it will be visible in all class